pom.xml

```diff
@@ -1526,17 +1525,6 @@
             </modules>
         </profile>
 
-        <profile>
-            <id>jdk18</id>
-            <activation>
-                <jdk>[18,24)</jdk>
-            </activation>
-            <properties>
-                <security.manager.props>
-                    -Djava.security.manager=allow
-                </security.manager.props>
-            </properties>
-        </profile>
```

Comment:
Note, there is no need for this which is why I removed it. By default, the wildfly-launcher will add the appropriate JVM arguments. I assume this was added originally as a workaround.